reference/4.0/Microsoft.PowerShell.Management/New-PSDrive.md

```diff
@@ -33,26 +33,22 @@ However, because temporary drives are known only to PowerShell, you cannot acces
 
 The following features are added to `New-PSDrive` in Windows PowerShell 3.0:
 
-* **Mapped network drives**
-
-  You can use the **Persist** parameter of `New-PSDrive` to create Windows mapped network drives.
+- Mapped network drives.
+  You can use the *Persist* parameter of `New-PSDrive` to create Windows mapped network drives.

```

Comment:
- Property, parameter, and class names should be **bold**

Reply:
Fixed